src/components/CheckboxInput/CheckboxInput.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { ChangeEvent, FocusEvent, FC } from 'react';
+import React, { ChangeEvent, FocusEvent } from 'react';
```

Comment:
Following the single import at the top convention, would we just have the React import at the top and then `React.ChangeEvent` and `React.FocusEvent` in the document?

Reply:
I think it might make what the component is depending on more obvious if we just do all of the imports at the top.

Reply:
seems like juan is in agreement so ill change it back